### PR TITLE
Add Trophy Support and add a sample for the trophy system

### PR DESCRIPTION
**Pull request by Backporter**

### CODE REVIEW

**Review by kiwidoggie — CHANGES_REQUESTED** [review]
Can you remove all of the extra sprx/prx binaries, they need to be compiled from source and copied in. Will review other changes later.

**Comment by Backporter** [comment]
totally forgot to remove them, doing that now.

File: samples/trophies/trophies/build.bat [hunk 1]
```diff
@@ -0,0 +1,80 @@
+SETLOCAL EnableDelayedExpansion
+
+Rem Package information
+set PKG_TITLE="OpenOrbis Trophy Sample"
+set PKG_VERSION="1.00"
+set PKG_TITLE_ID="BREW00080"
```
Comment (Cryptogenic) [inline samples/trophies/trophies/build.bat:6]: Title ID and content ID need updating, BREW00080 conflicts with audio wav sample. Recommend BREW00094. Makefile also needs to be written for samples for linux compatibility.

**Review by Cryptogenic — CHANGES_REQUESTED** [review]
Overall, need title ID/content ID updated to BREW00094, Makefile needs to be added, and icon0 should be updated (it's currently identical to audio wav sample).

**Comment by Backporter** [comment]
should be good now?

File: include/orbis/_types/rtc.h [hunk 2]
```diff
@@ -0,0 +1,3 @@
+typedef struct OrbisRtcTick {
+	uint64_t Ticks;
+} SceRtcTick;
```
Comment (bucanero) [inline include/orbis/_types/rtc.h:3]: I think this should be:

```suggestion
}OrbisRtcTick;
```

Comment (Backporter) [reply include/orbis/_types/rtc.h:3]: Right, missed that, fixing now.

**Pull request merged**